Sources/PortalSwift/Core/mpc/PortalMpc.swift

```diff
@@ -340,7 +340,49 @@ public class PortalMpc {
       }
     }
   }
+    
+    /// Uses the org and client backup shares to return the private key
+    ///  - Parameters:
+    ///    - cipherText: the cipherText of the client's backup share
+    ///    - method: The specific backup storage option.
+    ///    - orgShare: the stringified version of the organization's backup share
+    public func EjectWalletAndDiscontinueMPC(
+        cipherText: String,
+        method: BackupMethods.RawValue,
+        backupConfigs: BackupConfigs? = nil,
+        orgShare: String,
+        completion: @escaping (Result<String>) -> Void
+    ) {
+        if self.version != "v5" {
+            completion(Result (error: MpcError.recoverNoLongerSupported(message: "[PortalMpc] Recover is no longer supported for this version of MPC. Please use `version = v5`.")))
+        }
+
+        self.getBackupShare(cipherText: cipherText, method: method) { (result: Result<String>) in
+            guard result.error == nil else {
+                completion(Result (error: result.error!))
+                return
+            }
+            if let clientBackupShare = result.data{
+                // Call eject with clientBackupShare and orShare
+                let privateKey = self.mobile.MobileEjectWalletAndDiscontinueMPC(clientBackupShare, orgShare)
```

Comment:
is this just the private key or is it an object that contains an error object that you need to check to see if there was an mpc error

Reply:
Its a stringified object like `{"privateKey":"f1b4a6d055559708441dd2535e3a502fe8d8667fae8077ff2eec1f14476f316f","error":{"code":0,"prefix":"","error":"","message":""}}`

Reply:
Addressed